api/src/main/java/org/openmrs/module/kenyaemr/reporting/data/converter/definition/evaluator/ipt/DapsoneCotrimoxazoleDataEvaluator.java

```diff
@@ -36,9 +36,9 @@ public EvaluatedPersonData evaluate(PersonDataDefinition definition, EvaluationC
         EvaluatedPersonData c = new EvaluatedPersonData(definition, context);
 
         String qry = "select init.patient_id,if (init.patient_id = x.patient_id,\"Y\",\"N\") as on_ctx from kenyaemr_etl.etl_ipt_initiation init\n" +
-                "                      left outer join (select o.patient_id from openmrs.orders o inner join openmrs.drug_order do on o.order_id = do.order_id\n" +
-                "where o.concept_id in (105281,74250) group by o.patient_id) x on init.patient_id = x.patient_id\n" +
-                "where init.voided = 0;";
+                "                          left outer join (select o.patient_id from openmrs.orders o inner join openmrs.drug_order do on o.order_id = do.order_id\n" +
```

Comment:
not sure what this query intends to fetch. how is ctx and dapsone handled in ipt program? is there a form where these are recorded?